### PR TITLE
Fix: "scm.providerCountBadge" not working

### LINKED ISSUE #108957: "scm.providerCountBadge" not working
- VSCode Version: 1.50.1
- OS Version: Ubuntu 20.04

The "scm.providerCountBadge" setting no longer works (none of the 3 option).
If i remember correctly, this is responsible for the number on the right. But even if i set it to "hidden", it's still visible.
Currently it acts like it's always on "visible".

Screenshot from 2020-10-19 11-34-11

Does this issue occur when all extensions are disabled?: Yes

### PR DESCRIPTION
**Pull request by susiwen8**
This PR fixes #108957

### CODE REVIEW

**Comment by susiwen8** [comment]
@joaomoreno @eamodio Hi, Could you review this?

File: src/vs/base/browser/ui/countBadge/countBadge.ts [hunk 1]
```diff
@@ -47,7 +48,7 @@ export class CountBadge implements IThemable {
 		this.badgeBackground = this.options.badgeBackground;
 		this.badgeForeground = this.options.badgeForeground;
 		this.badgeBorder = this.options.badgeBorder;
-
+		this.container = container;
```
Comment (joaomoreno) [inline src/vs/base/browser/ui/countBadge/countBadge.ts:51]: The `CountBadge` does not own the `container`. It shouldn't do anything on it besides `appendChild`. We either set the `data-count` on `this.element` or we find a different way to fix this, if that doesn't work.

**Review by joaomoreno — CHANGES_REQUESTED** [review]

File: src/vs/base/browser/ui/countBadge/countBadge.ts (hunk 1 above)
Comment (susiwen8) [reply src/vs/base/browser/ui/countBadge/countBadge.ts]: @joaomoreno Hi, I have bind `count` to `this.element`, please review it.

**Comment by joaomoreno** [comment]
After closer inspection, I've closed the upstream issue: https://github.com/microsoft/vscode/issues/108957#issuecomment-767611338

**Comment by susiwen8** [comment]
@joaomoreno 
Here is video for this PR

https://user-images.githubusercontent.com/20318608/106373969-ca5b1b00-63b9-11eb-8999-1654acdc4b77.mp4

**Comment by susiwen8** [comment]
ping @joaomoreno.

**Comment by joaomoreno** [comment]
Replied in https://github.com/microsoft/vscode/issues/108957#issuecomment-772333074

**Pull request closed without merging**